web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -51,41 +30,82 @@
                   </div>
                 </template>
                 <span>
-                  Embargoed datasets are hidden from public access until a specific time period has
+                  Embargoed Dandisets are hidden from public access until a specific time period has
                   elapsed. Uploading data to the DANDI archive under embargo requires a relevant
                   NIH award number, and the data will be automatically published when the embargo
                   period expires.
                 </span>
               </v-tooltip>
             </template>
-          </v-checkbox>
-          <v-text-field
-            v-if="embargoed"
-            v-model="awardNumber"
-            label="Award number*"
-            hint="Provide an NIH award number for this embargoed dataset.
-                Note: this can be changed at any time and additional award
-                numbers can be added later."
-            persistent-hint
-            :counter="120"
-            :required="embargoed"
-            outlined
-            class="mt-4 shrink"
-            style="width: 20vw;"
-            :rules="awardNumberRules"
-          />
+          </v-switch>
+        </div>
+        <h1>Title</h1>
```

Comment:
The "vuetify way" to do this is to use the `text-h1` class instead of delegating styling choices to HTML with the `h#` elements.
```suggestion
        <div class="text-h1">Title</div>
```

Reply:
I did notice that, but also the `text-h1` class gives a _very_ different outcome than just using an `h1` element. I'll have to figure out what the right class to use is.

Reply:
`text-h4` does the trick. See f1cef20b8d9db90d0a24220dcbc721393749f8eb.